tensorflow/compiler/tf2tensorrt/convert/convert_nodes.h

```diff
@@ -522,6 +522,14 @@ class Converter {
                                const bool validation_only,
                                nvinfer1::ITensor** tensor);
 
+  // Helper function to add a squeeze op to the network.
```

Comment:
Either add an empty line after this or  make this a "full line". This is because we only allow partial line if the line is the last line of a paragraph.